src/components/Profile/TrackingSection.jsx

```diff
@@ -1,25 +1,46 @@
 import React from 'react'
 
+import { useQuery } from 'cozy-client'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 
 import Input from 'components/Input'
+import { buildSettingsInstanceQuery } from 'lib/queries'
+import { useMutation } from 'hooks/useMutation'
 
-export const TrackingSection = ({ instance, fields, onChange }) => {
+export const TrackingSection = () => {
   const { t } = useI18n()
+  const { mutate, isLoading, isSuccess, error } = useMutation()
+
+  const instanceQuery = buildSettingsInstanceQuery()
+  const { data: instance } = useQuery(
+    instanceQuery.definition,
+    instanceQuery.options
+  )
+
+  const handleChange = (_, value) => {
+    mutate({
+      _rev: instance.meta.rev,
+      ...instance,
+      attributes: {
+        ...instance.attributes,
+        tracking: value
+      }
+    })
+  }
 
   return (
     <Input
       name="tracking"
       type="checkbox"
       title={t('ProfileView.tracking.title')}
       label={t('ProfileView.tracking.label', {
-        version:
-          instance && instance.data.attributes.tos
-            ? `-${instance.data.attributes.tos}`
-            : '-201711'
+        version: instance?.tos ? `-${instance.tos}` : '-201711'
```

Comment:
No more data.attributes ?

Reply:
To work in real time, attributes are spread at the root of the object ([code](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/SettingsCollection.js#L13-L19)). Normally this should be the norm everywhere from what I understand, but migration is complex ([discussion about](https://github.com/cozy/cozy-client/issues/1118))

Reply:
Thanks for the explanation!